packages/core/admin/ee/admin/src/pages/SettingsPage/pages/AuditLogs/components/ComboboxFilter.tsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 
 import { Combobox, ComboboxOption } from '@strapi/design-system';
-import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 
-const ComboboxFilter = ({ value, options, onChange }) => {
+type ComboboxFilterProps = {
+  value?: string;
+  options?: { label: string; customValue: string }[];
+  onChange?: () => void;
```

Comment:
This doesn't seem to be right and it should receive at least one argument holding the value?